fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/service/ExternalServicesPropertiesReadPlatformServiceImpl.java

```diff
@@ -83,7 +83,7 @@ public SMTPCredentialsData extractData(final ResultSet rs) throws SQLException,
 
                 if (ExternalServicesConstants.SMTP_USERNAME.equalsIgnoreCase(name)) {
                     username = value;
-                } else if (ExternalServicesConstants.SMTP_PORT.equalsIgnoreCase(port)) {
+                } else if (ExternalServicesConstants.SMTP_PORT.equalsIgnoreCase(name)) {
```

Comment:
@ptuomola this change looks curious - is this intentional?

Reply:
sorry yeah no this is obviously right (it makes more sense looking at the full file then the diff here), nice catch!